ui/design-system-components/src/components/Entry/EntryCard/index.tsx

```diff
@@ -1,130 +1,216 @@
+            />
+          )}
+        </>
+        <Stack direction="row" spacing="gap-2" align="center" customStyle="shrink-0">
+          {entryData?.createdAt && !hidePublishTime && (
+            <Tooltip placement={'top'} content={formatDate(entryData?.createdAt, locale)}>
+              <Text customStyle="flex shrink-0 text(grey4 dark:grey7)">
+                {formatRelativeTime(entryData?.createdAt, locale)}
+              </Text>
+            </Tooltip>
+          )}
+          {entryData?.createdAt && (
+            <Tooltip
+              placement={'top'}
+              content={`${editedLabel} ${formatRelativeTime(entryData?.createdAt, locale)}`}
+            >
+              <Icon size="sm" type="PencilIcon" />
+            </Tooltip>
+          )}
+          {!entryData.active && (
+            <CardHeaderMenuDropdown
+              disabled={disableActions}
+              menuItems={
+                !entryData.author.isViewer
+                  ? [
+                      {
+                        icon: 'FlagIcon',
+                        handler: onEntryFlag,
+                        label: flagAsLabel,
+                        disabled: disableReporting,
+                      },
+                    ]
+                  : []
+              }
+              headerMenuExt={headerMenuExt}
+            />
+          )}
+        </Stack>
+      </Stack>
+      {isRemoved && (
+        <EntryCardRemoved
+          isAuthor={entryData.author.isViewer}
+          removedByAuthorLabel={removedByAuthorLabel}
+          removedByMeLabel={removedByMeLabel}
+        />
+      )}
+      {!isRemoved && (
+        <Button onClick={onContentClick} plain>
+          <Stack
+            customStyle={`px-4 max-h-[50rem] ${
+              scrollHiddenContent ? 'overflow-auto' : 'overflow-hidden'
+            } ${contentClickable ? 'cursor-pointer' : 'cursor-default'}`}
+            data-testid="entry-content"
+          >
+            {rest.itemType === EntityTypes.REFLECT ? (
+              <ReadOnlyEditor
+                content={rest.slateContent}
+                handleMentionClick={rest.onMentionClick}
+                handleTagClick={rest.onTagClick}
+                handleLinkClick={url => {
+                  rest.navigateTo?.({ getNavigationUrl: () => url });
+                }}
+              />
+            ) : (
+              rest.sortedContents.map(item => (
+                <Extension
+                  key={item.blockID}
+                  name={`${item.blockID}_content_block`}
+                  uiEvents={rest.uiEvents}
+                />
+              ))
+            )}
+          </Stack>
+        </Button>
+      )}
+      {!hideActionButtons && (
+        <CardActions
+          profileId={entryData.author.id}
+          repliesAnchorLink={repliesAnchorLink}
+          disableActions={disableActions}
+          isModerated={isModerated}
+          actionsRightExt={actionsRightExt}
+          onReflect={onReflect}
```

Comment:
isn't this prop (`onReflect`) used to navigate to a beam's page reflections section?

Reply:
If you saw the comment at the top of the card-actions component I have a todo to refactor the component :)

Reply:
oh, oky, sorry I missed that